package.json

```diff
@@ -40,32 +45,23 @@
   },
   "devDependencies": {
     "browserify": "^16.0.0",
+    "c8": "^6.0.1",
     "dtslint": "^1.0.2",
-    "nyc": "^14.0.0",
+    "microbundle": "^0.11.0",
     "prettier": "^1.0.0",
     "remark-cli": "^7.0.0",
     "remark-preset-wooorm": "^6.0.0",
     "tape": "^4.0.0",
-    "tinyify": "^2.5.1",
-    "typescript": "^3.0.0",
     "xo": "^0.25.0"
   },
   "scripts": {
-    "format": "remark . -qfo && prettier --write \"**/{*.js,*.ts}\" && xo --fix",
-    "build-bundle": "browserify index.js -s unified -o unified.js",
-    "build-mangle": "browserify index.js -s unified -p tinyify -o unified.min.js",
-    "build": "npm run build-bundle && npm run build-mangle",
+    "format": "remark . -qfo && prettier --write \"index.js\" \"types/**\" && xo --fix",
+    "build": "microbundle",
     "test-api": "node test",
-    "test-coverage": "nyc --reporter lcov tape test",
+    "test-coverage": "c8 --reporter=lcov --check-coverage --lines 100 --functions 100 --branches 100 tape test",
```

Comment:
also migrate coverage to `c8`, which leverages V8's native coverage capabilities, since the builds now are done on Node 10+
https://github.com/bcoe/c8

May open a separate PR just for this.

Reply:
C8 seems to be pretty young, unstable (see `pipelineParse`), and [include bugs](https://github.com/bcoe/c8/issues). What’s the benefit of switching?

Reply:
(oh and see: https://github.com/unifiedjs/unified/pull/76#discussion_r341409221)

Reply:
> C8 seems to be pretty young, unstable

Young-ish, 1.0 was released in Oct 25, 2017, over two years ago.
It is as stable as the coverage feature in v8.

> and includes bugs

:man_shrugging: so do Nyc and Istanbul
- https://github.com/istanbuljs/nyc/issues
- https://github.com/istanbuljs/istanbuljs/issues

> What’s the benefit of switching?

It is a native feature of Chrome/Node and should more accurately reflect runtime coverage, than the build-instrumentation based approach of Instanbul (some work may still be needed), as well has having the performance boost from running directly in the platform.